deploy/install/01-prerequisite/02-serviceaccount.yaml

```diff
@@ -53,6 +53,12 @@ rules:
 - apiGroups: ["metrics.k8s.io"]
   resources: ["pods", "nodes"]
   verbs: ["get", "list"]
+- apiGroups: ["apiregistration.k8s.io"]
+  resources: ["apiservices"]
+  verbs: ["*"]
+- apiGroups: ["admissionregistration.k8s.io"]
+  resources: ["mutatingwebhookconfigurations", "validatingwebhookconfigurations"]
+  verbs: ["*"]
```

Comment:
Do we need to add this for `longhorn-uninstall-role`?

Reply:
No, because we did not remove the `mutatingwebhookconfigurations` and `validatingwebhookconfigurations` after the uninstallation job.

Reply:
When/Where does Longhorn operate these 2 kinds of resources?

Reply:
https://github.com/longhorn/longhorn-manager/blob/87b7263df396dbdf9db9df00c51bc552767b816d/webhook/server/server.go#L96-L150

Reply:
Doesn't Longhorn clean them up during uninstallation?

Reply:
> Doesn't Longhorn clean them up during uninstallation?

Okay, I add the cleanup code in the uninstall job
- clean up the longhorn webhook Deployment
- clean up the ValidatingWebhookConfiguration and MutatingWebhookConfiguration.